src/Screens/Onboarding/Email.tsx

```diff
@@ -38,6 +39,8 @@ const Email = ({ navigation }: Props) => {
     });
   };
 
+  const canContinue = email === '' ? true : validateEmail(email);
```

Comment:
This is a very small comment about naming boolean variables. Please adjust this so that it begins with _is_ or _has_, like ===> _isValidEmail_, etc. :)